website/docs/secrets/spec/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
the whole spec directory is not showing when I run the docs locally. I think you need to add it to the sidebars file

Reply:
done!